source/mainnet/net/index.rst

```diff
@@ -53,6 +53,7 @@ index
    guides/developer-page
    nodes/node-requirements
    references/concordium-client
+   guides/run-local-chain
```

Comment:
When I run this locally, this hyperlink only shows up in the sidebar when on the `guides/developer-page` (i.e. it is not visible when I visit `node-requirements` or `references/concordium-client`). I am not sure what causes this, or if it is just an issue local to my system.

Reply:
I just built locally and it works fine for me. It shows up in the Develop on Concordium toctree as I would expect and it stays there when I navigate to other pages.

Reply:
That's great :-) I suspected that was the case.